fiftyone/core/annotation/generate_label_schemas.py

```
@@ -412,9 +415,11 @@ def _handle_str(collection, field_name, is_list, settings, scan_samples):
     try:
         if scan_samples and field_name != foac.FILEPATH:
             values = collection.distinct(field_name)
-    except OperationFailure:
-        # too many distinct values
-        pass
+    except OperationFailure as e:
+        # Likely too many distinct values
+        logger.debug(
+            f"Could not compute distinct values for field `{field_name}`: {e.details.get('errmsg')}"
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)